x/leverage/keeper/keeper_test.go

```diff
@@ -1273,36 +1188,6 @@ func (s *IntegrationTestSuite) TestBorrowAmountInvariant() {
 	s.Require().False(broken)
 }
 
-func (s *IntegrationTestSuite) TestBorrowAPYInvariant() {
```

Comment:
Note: Removed test as manually setting borrow APY is no longer possible (always derived instantly)